pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -480,14 +508,14 @@ public OffsetTime toOffsetTimeBin(byte[] bytes) throws PSQLException {
       return OffsetTime.MAX;
     }
 
-    final ParsedTimestamp ts = parseBackendTimestamp(s);
+    final ParsedTimestamp ts = parseBackendTimestamp(s.getBytes());
```

Comment:
Should `toOffsetTime(@PolyNull String s)` be adjusted to receive `byte[]` parameter, so it does not have to call `s.getBytes()` on every execution?